lib/block_view_validator.go

```diff
@@ -506,6 +528,41 @@ func (txindexMetadata *UnregisterAsValidatorTxindexMetadata) GetEncoderType() En
 	return EncoderTypeUnregisterAsValidatorTxindexMetadata
 }
 
+//
+// TYPES: UnjailValidatorTxindexMetadata
+//
+
+type UnjailValidatorTxindexMetadata struct {
+	ValidatorPublicKeyBase58Check string
+}
+
+func (txindexMetadata *UnjailValidatorTxindexMetadata) RawEncodeWithoutMetadata(blockHeight uint64, skipMetadata ...bool) []byte {
+	var data []byte
+	data = append(data, EncodeByteArray([]byte(txindexMetadata.ValidatorPublicKeyBase58Check))...)
```

Comment:
Isn't this a little unnecessary? won't it always just be the transactor? maybe we don't really need anything in here. idk